can/interfaces/socketcan/socketcan.py

```diff
@@ -642,6 +658,9 @@ def __init__(
             channel using :attr:`can.Message.channel`.
         :param receive_own_messages:
             If transmitted messages should also be received by this bus.
+        :param bool can_hardware_timestamps:
+            Use hardware timestamp for can messages if available instead of
+            the system timestamp.
```

Comment:
You could mention `SO_TIMESTAMPNS` and `SO_TIMESTAMPING` here. You could also mention possible pitfalls as @hartkopp mentioned

Reply:
done